### PR TITLE
feat: use Hadoop general way to export Classpath

### PR DESCRIPTION
**Pull request by zuston**
The original way will lack the hadoop config in classpath.

### CODE REVIEW

**Comment by zuston** [comment]
If OK, I think I can propose the same change to opendal/hdfs readme.

File: README.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ Please also make sure `HADOOP_HOME`, `LD_LIBRARY_PATH`, `CLASSPATH` is set corre
 ```shell
 export HADOOP_HOME=/path/to/hadoop
 export LD_LIBRARY_PATH=${JAVA_HOME}/lib/server:${LD_LIBRARY_PATH}
-export CLASSPATH=$(find $HADOOP_HOME -iname "*.jar" | xargs echo | tr ' ' ':')
+export CLASSPATH=$(${HADOOP_HOME}/bin/hadoop classpath --glob)
```
Comment (Xuanwo) [inline README.md:53]: In some use cases, `hadoop classpath --glob` is not correct. Maybe we can add this as an alternative?

**Review by Xuanwo — APPROVED** [review]
Thanks!

File: README.md (hunk 1 above)
Comment (zuston) [reply README.md:53]: Sorry, could you help indicate which case will be wrong? I have use this way in another project, which has been running in several years. But I have to say I always use this in CDH release.

**Pull request merged**